status-go-version.json

```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.159.6",
-    "commit-sha1": "f23cb4065bce9cff3cb8174a10bf9d8d49a4789f",
-    "src-sha256": "1yv24cqj2gnf2rgws39vc0j54l0dfxa6vc74iqcxgm50gd73gf6s"
+    "version": "status-mobile-16352",
+    "commit-sha1": "160094f641618de166de80c6ba22b19238e41d8c",
+    "src-sha256": "0c8hcqrr5y1kkva1xibz7dhd8crwbsypq6lc3n9mzl8dcvm98dxb"
```

Comment:
[This](https://github.com/status-im/status-go/pull/3673/files#diff-12e1567f20b885c074b2a1213664d43a77fbc87c34b5dc42cdb6d4526a86460dR5839-R5847) function on `status-go` counted the number of chats we should check for notification for, Without the `return true` statement I added to break cycle, the code added the muted chat list to the public chat list (Group chats and Public chats), Leading to push notifications to be send even if the chat is muted.